core/client/fs/src/main/java/alluxio/client/file/FileSystem.java

```diff
@@ -140,13 +149,13 @@ public static FileSystem create(FileSystemContext context) {
           LOG.debug("{}={} ({})", key.getName(), value, source);
         }
       }
-      Class fsClass = context.getClusterConf().getClass(PropertyKey.USER_FILESYSTEM_CLASS);
+      Class fsClass = conf.getClass(PropertyKey.USER_FILESYSTEM_CLASS);
```

Comment:
should this be done through `getClusterConf()`?

Reply:
We previously were trying to decouple filesystem creation from the file system context, but I don't think that is necessary anymore, I've reverted the changes. See: https://github.com/Alluxio/alluxio/pull/10907